frontend/src/stores/shortest-path/ShortestPathStateStore.ts

```diff
@@ -0,0 +1,25 @@
+import SimpleStore from '../SimpleStore';
+import { ShortestPathState } from './ShortestPathState';
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
why is this necessary?

Reply:
also add a short documentation what the store is responsible for

Reply:
>why is this necessary?   

I added an additional default export, such that this is no longer necessary.  

>also add a short documentation what the store is responsible for  

Done!